versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx

```diff
@@ -36,3 +38,11 @@ error: the server doesn't have a resource type "sliceconfigs"
 - Ensure you are on the controller cluster to run the commands: `kubectx -c`.
 - Export the configuration file using this command: `export KUBECONFIG=kubeslice/<path-to-the-kubeconfig-file>`.
 
+
+### Getting an Unverified Developer Error message on MAC
+You are unable to install the application on MAC due to the error message `Unverified Developer Error Message`.
```

Comment:
This sentence needs a bit rephrasing. 

You are unable to install because of the error is the scenario. 

due to is usually used when we describe the issue like no disk space or low memory etc. 

So, can you rephrase the above sentence as 'When you try to install kubeslice-cli on MacOS, if you get the `Unverified Developer Error Message`, then it means that you as a developer is not registered with Apple.` 

or something like that... 

Here the developer who installs gets the error, and we describe in second person; so the developer is you... Does it make sense?

Reply:
Updated the below sentence:
When you try to install kubeslice-cli on macOS, you get the `Unverified Developer Error Message`.
This error message appears when you try to install an application from a developer who has not registered with Apple.